contract/r/gnoswap/emission/utils.gno

```diff
@@ -90,3 +91,29 @@ func assertSumDistributionPct(pct01, pct02, pct03, pct04 uint64) {
 		))
 	}
 }
+
+func formatUint(v interface{}) string {
+	switch v := v.(type) {
+	case uint8:
+		return strconv.FormatUint(uint64(v), 10)
+	case uint32:
+		return strconv.FormatUint(uint64(v), 10)
+	case uint64:
+		return strconv.FormatUint(v, 10)
+	default:
+		panic(ufmt.Sprintf("invalid type: %T", v))
+	}
+}
+
+func formatInt(v interface{}) string {
+	switch v := v.(type) {
+	case int32:
+		return strconv.FormatInt(int64(v), 10)
+	case int64:
+		return strconv.FormatInt(v, 10)
+	case int:
+		return strconv.Itoa(v)
+	default:
+		panic(ufmt.Sprintf("invalid type: %T", v))
+	}
+}
```

Comment:
Seems like you're willing to format numeric values, in this case please handle all size
`8` `16` `32` `64`

Reply:
I have only registered the types that are currently being used in the gnoswap repository